src/interface/widgets/widget.cpp

```diff
@@ -102,6 +105,20 @@ void Visualizer::paintEvent(QPaintEvent *event) {
     Input::clearDrawings();
 
     if (showBallPlacementMarker) drawBallPlacementTarget(painter);
+
+    /* Ball dragging using middle mouse button. Hold it to drag the ball */
+    if (middle_mouse_pressed and not SETTINGS.isSerialMode()) {
```

Comment:
@emielsteerneman I see you use the SETTINGS.isSerialMode(), but [another PR](https://github.com/RoboTeamTwente/roboteam_ai/pull/1320/files) has removed this function and changed it to SETTINGS.getRobotHubMode() instead. The thing that worries me is that github does not say your branch is out of date, so I do not know what will happen when this gets merged

Reply:
That makes sense right? `isSerialMode()` was not replaced in `widget.cpp` in that other PR, since `widget.cpp` didn't even have `isSerialMode()` in that PR, because this PR wasn't merged at that moment. Could I please ask you to fix this issue?

Reply:
fixed this